meilisearch/task.py

```diff
@@ -27,6 +27,53 @@ def __init__(self, config: Config):
         self.config = config
         self.http = HttpRequests(config)
 
+    def get_batches(self, parameters: Optional[MutableMapping[str, Any]] = None) -> BatchResults:
+        """Get all task batches.
+
+        Parameters
+        ----------
+        parameters (optional):
+            parameters accepted by the get batches route: https://www.meilisearch.com/docs/reference/api/batches#get-batches.
+
+        Returns
+        -------
+        batch:
+            BatchResults instance contining limit, from, next and results containing a list of all batches.
+
+        Raises
+        ------
+        MeilisearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+        """
+        if parameters is None:
+            parameters = {}
+        for param in parameters:
+            if isinstance(parameters[param], list):
```

Comment:
`list` is the most likely type we will get here, but just for safety since the types aren't specified anywhere lets also check for a `tuple` since it will work also.

```suggestion
            if isinstance(parameters[param], (list, tuple}):
```

Reply:
Done! Is it worth going through other examples of this and updating them as well?

Reply:
I think the chance of getting a tuple is low (and we have never had a bug report on this), but it couldn't hurt to add this in other places also to be safe.